echo/frontend/src/components/conversation/ConversationAccordion.tsx

```
@@ -780,13 +780,9 @@ export const ConversationAccordion = ({ projectId }: { projectId: string }) => {
             <Trans>Conversations</Trans>
           </Title>
 
-          <Tooltip label={`Upload conversations`}>
-            <div>
-              <UploadConversationDropzone projectId={projectId}>
-                <Icons.Plus stroke="black" fill="black" />
-              </UploadConversationDropzone>
-            </div>
-          </Tooltip>
+          <div onClick={(e) => e.stopPropagation()}>
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)